README.md

```diff
@@ -53,8 +53,8 @@ jenkins-plugin-cli --plugin-file /your/path/to/plugins.txt --plugins delivery-pi
 * `--view-all-security-warnings`: (optional) Set to true to show all plugins that have security warnings.
 * `--available-updates`: (optional) Set to true to show if any requested plugins have newer versions available. If a Jenkins version-specific update center is available, the latest plugin version will be determined based on that update center's data.
 * `--output {stdout,yaml,txt}`: (optional) Format to output plugin updates file in, stdout is the default.
-* `--latest false`: (optional) Set to false to download the minimum required version of all dependencies.
-* `--latest-specified`: (optional) (advanced) Set to true to download latest dependencies of any plugin that is requested to have the latest version. All other plugin dependency versions are determined by the update center metadata or the plugin MANIFEST.MF.
+* `--latest false`: (optional) Set to false to download the minimum required version of all dependencies unless some specific version is explicitly requested. (default: true)
```

Comment:
Can you revert these docs updates, I don't think we need to add this caveat. 
This PR fixes the behaviour to be as expected rather than a bug.

Reply:
done